edx_repo_tools/conventional_commits/extra.txt

```diff
@@ -49,7 +49,7 @@ python-dateutil==2.8.2
     # via
     #   matplotlib
     #   pandas
-pytz==2022.2.1
+pytz==2022.4
```

Comment:
No breaking change affecting the current implementation https://github.com/stub42/pytz/commit/8eeefc360913ce56f8759c33b31e00542a0d3078